test/models/queued_email_test.rb

```diff
@@ -8,10 +8,11 @@ def test_send_email
     assert_not(email.send_email)
   end
 
-  def test_send_email_no_to_from
-    email = QueuedEmail.new
-    assert_not(email.send_email)
-  end
+  # Actually it does now send with no_to_from, if there's also no user
+  # def test_send_email_no_to_from
+  #   email = QueuedEmail.new
+  #   assert_not(email.send_email)
+  # end
```

Comment:
Want to double-check that it's ok to eliminate this test, now that the mailer will actually send without a `to` and a `from`

Reply:
I can't follow all of that!  But this one, yes, I think this test is super unimportant, so it's okay to delete it.

Reply:
Will delete. 

I had multiple threads going here in the github PR -- that might have gotten confusing on email, sorry